src/components/Tabs/verticalTabWLine.css

```diff
@@ -0,0 +1,64 @@
+.lil-vertical-tabs{
+    @apply lil-list-none lil-inline-block lil-overflow-x-auto;
+}
+.lil-vertical-tabs .lil-vertical-tab{
+    min-height:100px;
+    @apply lil-opacity-50 lil-flex lil-whitespace-normal lil-cursor-pointer lil-outline-none;
+}
+.lil-vertical-tab .lil-vertical-tab-container{
+    @apply lil-flex lil-items-center lil-w-full  lil-border-l-xl lil-border-font-200 lil-flex-wrap;
+} 
+.lil-vertical-tab-container .lil-vertical-tab-content{
+    @apply lil-border-b-sm lil-border-font-200 lil-flex lil-items-center lil-h-full lil-justify-between lil-w-10/12 lil-py-5 lil-flex-wrap;
+}
+.lil-vertical-tab-container .lil-vertical-tab-line{
+    border-bottom: none;
+    @apply lil-flex lil-w-1/12 lil-h-full;
+}
+
+.lil-vertical-tab .lil-vertical-tab-heading{
+    @apply lil-text-s-20 lil-text-font-500 lil-w-full;
+}
+
+.lil-vertical-tab .lil-vertical-tab-arrow{
+    @apply lil-text-s-24 lil-text-font-500 lil-h-full lil-flex lil-items-center lil-w-1/12 lil-py-5 lil-border-b-sm lil-border-font-200;
+}
+
+.lil-vertical-tab .lil-vertical-tab-writeup{
+    @apply lil-hidden lil-w-full;
+}
+/*Defining the behaviour for tab when active*/
+.lil-vertical-tabs .lil-vertical-tab-active{
+    @apply lil-opacity-100;
+}
+
+.lil-vertical-tab-active .lil-vertical-tab-heading{
+    @apply lil-text-s-24 lil-font-bold;
+}
+.lil-vertical-tab-active .lil-vertical-tab-container{
+    @apply lil-items-start lil-border-opacity-50;
+}
+.lil-vertical-tab-active .lil-vertical-tab-line{
+    /*These values are to allign the line and give proper height*/
+    margin-left: -4px;
+    height: 80%;
+    @apply lil-border-l-xl lil-border-blue-350;
+}
+
+.lil-vertical-tab-active .lil-vertical-tab-content{
+    @apply lil-border-opacity-50 lil-w-11/12;
+}
+
+.lil-vertical-tab-active .lil-vertical-tab-arrow{
+    @apply lil-hidden;
+}
+
+.lil-vertical-tab-active .lil-vertical-tab-writeup{
+    @apply lil-flex lil-text-base lil-text-font-200 lil-py-3 lil-visible;
```

Comment:
`lil-visible` this is for ?

Reply:
in acitve state of tab, the description is visible in CTA where as it is hidden in the other case